redis_aamks/worker/worker.py

```diff
@@ -33,14 +33,14 @@ def redis_queue_pop(self, db):
         # the `b` in `brpop` indicates this is a blocking call (waits until an item becomes available)
         _, message = db.brpop(config.redis_queue_name)
         message_json = loads(message)
-        logger.debug(f'pop from head of queue \n{message_json["data"]}')
+        # logger.debug(f'pop from head of queue \n{message_json["data"]}')
```

Comment:
maybe logger can stay

Reply:
Ok, I restore logger. Logger doesnt work on our test environment (83,84 machines) - there is a problem with environment variables. There will be an exception when reading environment variables.